doc/expressionLanguage.md

```diff
@@ -682,51 +684,29 @@ two possible types of expressions: Integer (arithmetic operations) or Strings.
 | `${trim(@active)`       | `null`           | None    |
 | `${trim(@consumption)}` | `0.44`           | Float   |
 
-[1]:
```

Comment:
Changes from this line on should be reverted. Note that they break some links:

![imagen](https://user-images.githubusercontent.com/1534240/150120978-72e21386-2c93-40c0-8b57-b070b193011a.png)

(Compare with same at `master` branch)